advanced_functionality/multi_model_sklearn_home_value/sklearn_multi_model_endpoint_home_value.ipynb

```diff
@@ -2,7 +2,17 @@
  "cells": [
```

Comment:
<p>You deleted the last <code>predictor.delete_model()</code>  - I think it works when you delete model, and then delete endpoint.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2607/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>